versions/c-/catch2.json

```
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "900833643e0eb468d957b4c46f46ba6647e283ff",
+      "git-tree": "4df380019e5c3a0e450798db43d9ce70906b7aad",
+      "version-semver": "2.13.7",
+      "port-version": 1
+    },
+    {
+      "git-tree": "eea1021d4b3c4e41919b5e96a4372604e5837979",
```

```suggestion
      "git-tree": "900833643e0eb468d957b4c46f46ba6647e283ff",
```
This seems not right. It should not change. The new item should be added to the top of original ones. Could you please revert this change and use` ./vcpkg x-add-version --overwrite-version catch2` to update this?

---

I run command one step by step as below, am i right ?

1. change ports/catch2/portfile.cmake
2. add port-version on ports/catch2/vcpkg.json
3. vcpkg format-manifest ports/*/vcpkg.json
4. vcpkg x-add-version --overwrite-version catch2
5. git commit -m "update baseline & version" ports/ versions
6. git push

---

You should commit the changes(in ports/catch2/portfile.cmake and ports/catch2/vcpkg.json) first, then run `vcpkg x-add-version --overwrite-version catch2` and commit the changes about versions. Last, git push the commits. Or you can push the commit after you commit the one.

---

I redo command by adding "git commit" between step 3 and 4 , and push by force.  thx for your reply